client/app/routes/resources/resources.html

```diff
@@ -1,11 +1,51 @@
 <div ng-include="'components/navbar/navbar.html'"></div>
 <div ng-include="'components/sideNav/sideNav.html'" ></div>
-
+<head> 
+<base target="_blank">
+</head>
 <div class="page">
 	<section class="container">
 
-		This is the resources view. More to come!
+		Here are some resources. More to come!
+ <img class="resources-image" src="assets/images/resources7text.png"/>
+ <div class="vision-board">
```

Comment:
None of these `<div>` tags have ending tags, and should be `<ul>` tags if you are going to but `<li>` tags in them